src/parser/jobs/drk/modules/GCDs.js

```diff
@@ -95,17 +103,18 @@ export default class GCDs extends Module {
 		}
 	}
 
-	onComplete() {
+	_onComplete() {
 		//dropped combo chain
 		if (this._GCDChainDrops.length > 0) {
```

Comment:
I'd pull this block out of the conditional since the TieredSuggestion will handle display for you anyway.